docs/topics/Permissions.md

```diff
@@ -163,19 +163,28 @@ Roles represent a set of permissions attached to a group of users. Roles have un
 
 ###### Role Structure
 
-| Field           | Type      | Description                                      |
-| --------------- | --------- | ------------------------------------------------ |
-| id              | snowflake | role id                                          |
-| name            | string    | role name                                        |
-| color           | integer   | integer representation of hexadecimal color code |
-| hoist           | boolean   | if this role is pinned in the user listing       |
-| position        | integer   | position of this role                            |
-| permissions     | string    | permission bit set                               |
-| managed         | boolean   | whether this role is managed by an integration   |
-| mentionable     | boolean   | whether this role is mentionable                 |
+| Field       | Type                                                             | Description                                      |
+| ----------- | ---------------------------------------------------------------- | ------------------------------------------------ |
+| id          | snowflake                                                        | role id                                          |
+| name        | string                                                           | role name                                        |
+| color       | integer                                                          | integer representation of hexadecimal color code |
+| hoist       | boolean                                                          | if this role is pinned in the user listing       |
+| position    | integer                                                          | position of this role                            |
+| permissions | string                                                           | permission bit set                               |
+| managed     | boolean                                                          | whether this role is managed by an integration   |
+| mentionable | boolean                                                          | whether this role is mentionable                 |
+| tags?       | [role tags](#DOCS_TOPICS_PERMISSIONS/role-tags-structure) object | the tags this role has                           |
```

Comment:
I think the right link to this is `#DOCS_TOPICS_PERMISSIONS/role-object-role-tags-structure`